components/compliance-service/inspec/cli.go

```diff
@@ -250,6 +250,11 @@ func getInspecError(serr string, err error, target *TargetConfig, timeout time.D
 		return NewInspecError(UNREACHABLE_HOST,
 			"Failed to connect to "+target.Hostname+", host is unreachable.")
 	}
+	connTimeoutErr := "Net::SSH::ConnectionTimeout"
+	if strings.Index(serr, connTimeoutErr) >= 0 {
```

Comment:
this should help with the reports that there are a lot of "unknown error" error msgs